relay-server/src/actors/processor.rs

```diff
@@ -522,13 +529,15 @@ impl FromMessage<RateLimitFlushBuckets> for EnvelopeProcessor {
 /// This service handles messages in a worker pool with configurable concurrency.
 #[derive(Clone)]
 pub struct EnvelopeProcessorService {
+    global_config: Arc<GlobalConfig>,
     inner: Arc<InnerProcessor>,
 }
 
 struct InnerProcessor {
     config: Arc<Config>,
     envelope_manager: Addr<EnvelopeManager>,
     project_cache: Addr<ProjectCache>,
+    global_config: Addr<GlobalConfigInterface>,
```

Comment:
Does this really need to reside in the inner processor? This should be outside, as it's not needed to process envelopes.

Reply:
I thought we wanted to put everything that we don't change into the inner, so that we only need to clone two Arcs, and we don't need to change the global_config addr